content/en/about/integration.md

```diff
@@ -0,0 +1,57 @@
+---
+type: docs
+category: About Sigstore
```

Comment:
I don't think "About Sigstore" is the right category for this page. Would it make sense to have a top-level category like Signing and Verifying? Or combine this with System Configuration?

Reply:
moved to system config